docs/package.json

```diff
@@ -28,7 +28,7 @@
     "common-tags": "^1.8.0",
     "gatsby": "^1.9.279",
     "gatsby-link": "^1.6.45",
-    "gatsby-plugin-sharp": "^2.0.12",
+    "gatsby-plugin-sharp": "^1.6.48",
```

Comment:
@ndelangen why downgrade?

Reply:
I had some issues, I think the gatsby-plugin-sharp v2 is for gatsby v2